index.js

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from './modules/luxon.min.js';
+import Display from './modules/display.js';
+import Books from './modules/list.js';
+
+const books = new Books();
+const pages = ['#list', '#add', '#contact'];
+const menu = document.querySelector('.header-menu');
+const time = document.querySelector('.header time');
+const links = document.querySelectorAll('.header-menu a');
+const contents = document.querySelectorAll('.content');
+const list = document.querySelector('.content-list');
+const form = document.querySelector('.content-form');
+
+function displayBooks() {
+  if (books.list.length === 0) {
+    list.innerHTML = '<h2>No available books</h2>';
+  } else {
+    list.innerHTML = '';
+    list.appendChild(Display.list(books.list));
+  }
+}
+function contentEvent(event) {
+  event.preventDefault();
+  if (event.target.hash !== '') {
+    Display.page(event.target.hash);
+  }
+}
+function addEvent(event) {
+  event.preventDefault();
+  books.add(form.title.value, form.author.value);
+  displayBooks();
+  form.reset();
+}
+function removeEvent(event) {
+  if (event.target.id !== '') {
+    books.remove(event.target.id);
+    displayBooks();
+  }
+}
```

Comment:
- [ ] One of the requirements for this project is to use the **_arrow syntax_** to create functions instead of the conventional way of creating functions. Remember that **_arrow syntax_** is one of the major introductions in _**ES6**_. Since you are practising **_ES6_**, it would be nice to use **_arrow syntax_**. Kindly convert your functions to use arrow syntax. 👍 😎